tezos/protocols.go

```diff
@@ -110,8 +110,8 @@ var (
 			{PtLimaPt, 0, 1654785, 2162688, 404, 3, 4096, 256},  // v15
 			{PtMumbai, 0, 2162689, 2957312, 528, 3, 8192, 512},  // v16
 			{PtNairobi, 0, 2957313, 5316608, 625, 3, 8192, 512}, // v17
-			{Proxford, 0, 5316609, -1, 913, 3, 8192, 512},       // v18
-			// {PtParisB, 0, 5316609, -1, 913, 3, 8192, 8192},       // v19
+			{Proxford, 0, 5316609, 6422528, 913, 3, 8192, 512},  // v18
+			{PtParisB, 0, 6422529, -1, 1048, 3, 12288, 0},       // v19
```

Comment:
BlocksPerSnapshot cannot be 0. This will result in divide by zero error in params.go

Reply:
Talked to TzKT on why their value is eq to blocksPerCycle now that snapshots don't exist and they said, "because there's no snapshot, as a workaround we just imagine like there's one per cycle"

Reply:
updated